### ISSUE TITLE
feature: add traffic obfuscation

### ISSUE THREAD

**Issue by donBarbos** [notthebee/ansible-easy-vpn#225]
**Is your feature request related to a problem? Please describe.**
VPN protocols are easily detected (using DPI), so they're easily blocked and don't help mush against real (strict) censorship

**Describe the solution you'd like**
I would like to add a traffic obfuscation function, for example, through `Stunnel` or `Shadowsocks` or `SSTP` or something else

**Additional context**
often blocked by wireguard in russia and china, it's not difficult for them to block in a specific region or for a specific internet service provider.

**Comment by notthebee** [comment]
Duplicate of https://github.com/notthebee/ansible-easy-vpn/issues/145
